### DISCUSSION TITLE
KI zum erstellen von Programmen

### DISCUSSION THREAD

**Discussion by bluhminga** [micropython/micropython#11233] [Ideas]
Hallo,

ich habes was interessantes gefunden.

https://chat.openai.com/chat

Einfach registrieren und z.B.
"Ds18b20 auslesen" eingeben.

Echt interessant. Kranke Scheisse 😃😃😃😃😃

**Comment by sosi-deadeye** [comment]
Hello,

> 
> I found something interesting.
> 
> https://chat.openai.com/chat
> 
> Just register and enter e.g.
> "Read out Ds18b20".
> 
> Really interesting. Sick shit smileysmileysmileysmiley

I saw it on Reddit...
Some questioners tried ChatGPT3, but the result is so bad, that humans still have to help out. ChatGPT3 is a stochastic parrot.
